src/lib/core/CHIPCircularTLVBuffer.h

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2021 Project CHIP Authors
```

Comment:
supernit: should this be 2020-2021 like the nest labs below?

Reply:
Good point; will fix.